src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSourceUnitTest.java

```diff
@@ -98,6 +102,34 @@ public void testPartitionSizing(){
         Assert.assertEquals(smallPartitions.partitions().size(), 2);
     }
 
+    @Test
+    public void testReadFromFileAndHDFS() throws IOException {
+        final File bam = new File( getToolTestDataDir(), "reads.bam");
```

Comment:
Any name that makes it clear what the test data is being used for -- eg., `hdfs_test.bam`